Classes/Form/FormDataProvider/TcaTypesShowitemMaskBeLayoutFields.php

```diff
@@ -19,7 +19,7 @@ public function addData(array $result)
             $conditionMatcher = GeneralUtility::makeInstance(ConditionMatcher::class, null, $result['vanillaUid'], $result['rootline']);
             foreach ($json['pages']['elements'] as $element) {
                 $key = $element['key'];
-                if ($conditionMatcher->match("[maskBeLayout($key)]")) {
+                if ($conditionMatcher->match("[maskBeLayout('".$key."')]")) {
```

Comment:
Can you add blank spaces around the dot or use interpolation (think its prettier), thanks!